.github/workflows/version_check.yml

```diff
@@ -55,56 +73,67 @@ jobs:
         id: get_curr_ocp_version
         run: |
           OCP_VERSION=$(./oc version -o json | jq '.openshiftVersion')
+          OCP_VERSION=$(sed -e 's/^"//' -e 's/"$//' <<< $OCP_VERSION)
           printf "[INFO] Current OCP Version: %s\n" ${OCP_VERSION}
           echo "::set-output name=curr_ocp_version::${OCP_VERSION}"
         shell: bash
 
       - name: Checkout software-version branch
-        if: steps.check_repo.outputs.run-job == 'true'
+        if: steps.check_repo.outputs.check-version == 'true'
         uses: actions/checkout@v2
         with:
           ref: "software-version"
-          repository: "openshift-helm-charts/charts"
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
Will it be running with event pull_request also?

Reply:
Good sport. update to use ${{ github.repository }}. Interestingly the above did not cause failures because it evaluates to null and the default value works.